content/docs/user-guide/components/reference/shape/spline.md

```diff
@@ -1,75 +1,91 @@
 ---
-title: Spline component
+title: Spline Component
 linktitle: Spline
 description: ' Open 3D Engine (O3DE) Spline component reference. '
 weight: 100
 ---
 
 
 
-The **Spline** component creates a line that is 4 meters long, with 4 points. The point positions are defined in the local space of the entity. The spline's length, segments, and shape can be defined using the component's **Spline Type** property and **Edit** functionality. Splines are curves that connect two or more points and can be used as paths for animated entities or as a backbone for components such as the [Tube Shape](/docs/user-guide/components/reference/shape/tube-shape/) component.
+The **Spline** component creates a line that is 8 meters long, with 4 points. The point positions are defined in the local space of the entity. The spline's length, segments, and shape can be defined using the component's **Spline Type** property and **Edit** functionality. Splines are curves that connect two or more points and can be used as paths for animated entities or as a backbone for components such as the [Tube Shape](/docs/user-guide/components/reference/shape/tube-shape/) component.
 
-## Provider ##
+## Provider
 
 [O3DE Core (LmbrCentral) Gem](/docs/user-guide/gems/reference/o3de-core)
 
-## Base properties ##
+## Spline properties
 
-![Spline component properties](/images/user-guide/components/reference/shape/spline-component-ui-01.png)
+{{< tabs name="spline-component-ui" >}}
+{{% tab name="Linear Spline" %}}
+
+![Linear Spline](/images/user-guide/components/reference/shape/linear-spline-component-ui-01.png)
+
+{{% /tab %}}
+{{% tab name="Bezier Spline" %}}
+
+![Bezier Spline](/images/user-guide/components/reference/shape/bezier-spline-component-ui-01.png)
+
+{{% /tab %}}
+{{% tab name="Catmull-Rom Spline" %}}
+
+![Catmull-Rom Spline](/images/user-guide/components/reference/shape/catmull-rom-spline-component-ui-01.png)
+
+{{% /tab %}}
+{{< /tabs >}}
```

Comment:
Going to suggest moving the rest of this component property ref into the tabs for consistency. Yes, that means you'll need to copy-paste the standard table three times, but improves the layout of the page otherwise. Duplicate text in tabs is something we need to resolve, especially for platform-specific tabs and text